docs/host/host.md

```diff
@@ -33,30 +33,122 @@ Cordoning a node marks it as unschedulable. This feature is useful for performin
 
-Deleting a node is done in two phases:
+:::caution
 
-1. Delete the node from Harvester
-    - Go to the **Hosts** page
-    - On the node you want to modify, click **⋮ > Delete**
+Before removing a node from a Harvester cluster, determine if the remaining nodes have enough computing and storage resources to take on the workload from the node to be removed. Check the following:
 
-2. Uninstall RKE2 from the node
-    - Login to the node as root
-    - Run `rke2-uninstall.sh` to delete the whole RKE2 service.
+- Current resource usage in the cluster (on the **Hosts** screen of the Harvester UI)
+- Ability of the remaining nodes to maintain enough replicas for all volumes
 
-:::caution
+If the remaining nodes do not have enough resources, VMs might fail to migrate and volumes might degrade when you remove a node.
+
+:::
+
+### 1. Check if the node can be removed from the cluster.
+
+You can safely remove a control plane node depending on the quantity and availability of other nodes in the cluster.
+
+- The cluster has three control plane nodes and one or more worker nodes.
+
+  When you remove a control plane node, a worker node will be promoted to control plane node. Harvester v1.3.0 allows you to assign a role to each node that joins a cluster. In earlier Harvester versions, worker nodes were randomly selected for promotion. If you prefer to promote specific nodes, please see [Role Management](./host.md#role-management) and [Harvester Configuration](../install/harvester-configuration.md#installrole) for more information.
+
+- The cluster has three control plane nodes and no worker nodes.
+
+  You must add a new node to the cluster before removing a control plane node. This ensures that the cluster always has three control plane nodes and that a quorum can be formed even if one control plane node fails.
+
+- The cluster has only two control plane nodes and no worker nodes.
+
+  Removing a control plane node in this situation is not recommended because etcd data is not replicated in a single-node cluster. Failure of a single node can cause etcd to lose its quorum and shut the cluster down.
+
+
+### 2. Check the status of volumes.
+
+1. Access the [embedded Longhorn UI](../troubleshooting/harvester.md#access-embedded-rancher-and-longhorn-dashboards).
+
+1. Go to the **Volume** screen.
+
+1. Verify that the state of all volumes is **Healthy**.
+
+### 3. Evict replicas from the node to be removed.
+
+1. Access the [embedded Longhorn UI](../troubleshooting/harvester.md#access-embedded-rancher-and-longhorn-dashboards).
+
+1. Go to the **Node** screen.
+1. Select the node that you want to remove, click the icon in the **Operation** column, and then select **Edit node and disks**.
+
+1. Configure the following settings:
+
+  - **Node Scheduling**: Select **Disable**.
+  - **Evict Requested**" Select **True**.
+
+1. Click **Save**.
+
+1. Go back to the **Node** screen and verify that **Replicas** value for the node to be removed is **0**.
```

Comment:
Do we need an indent here?

Reply:
nice spot: it looks OK in the preview https://65f2b02337ffb294638e3ef6--harvester-preview.netlify.app/v1.3/host/#3-evict-replicas-from-the-node-to-be-removed